hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/BaseFreonGenerator.java

```diff
@@ -290,6 +290,10 @@ public void init() {
       //replace environment variables to support multi-node execution
       prefix = resolvePrefix(prefix);
     }
+    if (!allowDuration()) {
```

Comment:
Sorry, I missed a detail in my previous suggestion: we need to check if `--duration` is set by the user, otherwise the warning will always appear.

```suggestion
    if (duration != null && !allowDuration()) {
```